brian2/codegen/runtime/weave_rt/templates/synapses_create_generator.cpp

```diff
@@ -66,7 +66,7 @@
             {% endif %}
         }
         {% if iterator_func=='range' %}
-        for(int {{iteration_variable}}=_uiter_low; {{iteration_variable}}<_uiter_high; {{iteration_variable}}+=_uiter_step)
+        for(long {{iteration_variable}}=_uiter_low; {{iteration_variable}}<_uiter_high; {{iteration_variable}}+=_uiter_step)
```

Comment:
What's the difference between long and int here? Used a few times throughout. I think long is not guaranteed to be 64 bit?

Reply:
Yes, I stopped short of fixing this in any fundamental way. It just seemed odd that we use `int` here when `_uiter_low` was defined as a `long` earlier.